samples/SqlExtensionSamples/GlobalSuppressions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+// These suppression rules override parameters required by functions binding that cannot be converted to discard variables per issue: https://github.com/Azure/azure-functions-dotnet-worker/issues/323
+[assembly: SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "Unused parameter is required by functions binding", Scope = "member", Target = "~M:SqlExtensionSamples.AddProductsArray.Run(Microsoft.AspNetCore.Http.HttpRequest,SqlExtensionSamples.ProductUtilities.Product[]@)~Microsoft.AspNetCore.Mvc.IActionResult")]
```

Comment:
How did you generate these suppressions?

Reply:
Visual Studio :)

Reply:
![image](https://user-images.githubusercontent.com/13544267/135691521-7cab8ad9-7c35-48ed-bbce-27916d4d9824.png)